gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/components/api-runtime-logs-quick-filters/pipes/chip-value.pipe.ts

```diff
@@ -17,18 +17,26 @@ import { Pipe, PipeTransform } from '@angular/core';
 
 import { MultiFilter, SimpleFilter } from '../../../models';
 
+type ArrayFilter = MultiFilter | string[] | Set<number>;
+
 @Pipe({ name: 'chipValue' })
 export class ChipValuePipe implements PipeTransform {
-  transform(filter: SimpleFilter | MultiFilter | string[]): string {
-    return Array.isArray(filter)
-      ? filter
-          .map((value) => {
-            if (value.label) {
-              return value.label;
-            }
-            return value;
-          })
-          ?.join(', ')
-      : filter.label;
+  transform(filter: SimpleFilter | ArrayFilter): string {
+    if (Array.isArray(filter)) {
+      return filter
+        .map((value) => {
+          if (value.label) {
+            return value.label;
+          }
+          return value;
```

Comment:
maybe add a comment to understand why it can be `value.label` or `value`
It's clear reviewing the PR but we may forget later.